nanolib/include/conf.h

```diff
@@ -16,9 +16,13 @@
 	"{start|restart [-conf <path>] [-url <url>] [-daemon] [-tq_thread "   \
 	"<num>] [-max_tq_thread <num>] [-parallel <num>] [-qos_duration "     \
 	"<num>]}|stop}\n"                                                     \
-	"  -conf <path>          the path of a specified configuration file " \
+	"  -conf <path>          the path of a specified nanomq "             \
+	"configuration file "                                                 \
 	"\n"                                                                  \
-	"  -url <url>            the format of 'tcp://ip_addr:host'\n"        \
```

Comment:
please keep url feature

Reply:
and reminder

Reply:
It was moved to conf.h:24 :"  -url <url>            the format of 'broker+tcp://ip_addr:host' "